### DISCUSSION TITLE
File explorer disappears when creating new note from template

### DISCUSSION THREAD

**Discussion by yummy527** [SilentVoid13/Templater#706] [Help]
There’s a gif of the issue here: https://i.imgur.com/baeLKPP.gif

    Operating system: arch linux

    Debug info:
    SYSTEM INFO:
    Obsidian version: v0.15.6
    Installer version: v0.15.6
    Operating system: #1 SMP PREEMPT_DYNAMIC Fri, 15 Jul 2022 15:33:02 +0000 5.18.12-arch1-1
    Login status: not logged in
    Insider build toggle: off
    Live preview: on
    Legacy editor: off
    Base theme: adapt to system
    Community theme: none
    Snippets enabled: 0
    Safe mode: off
    Plugins installed: 21
    Plugins enabled: 16
    1: Advanced Tables
    2: Outliner
    3: Zoom
    4: Calendar
    5: Periodic Notes
    6: Templater
    7: Obsidian TODO | Text-based GTD
    8: Kanban
    9: Reminder
    10: Tasks
    11: Image Caption
    12: Local images
    13: Dataview
    14: Emoji Toolbar
    15: Excalidraw
    16: Highlightr

**Comment by AB1908** [comment]
Duplicate of #691?
